Code/Editor/CMakeLists.txt

```diff
@@ -276,6 +276,11 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED)
             Legacy::EditorLib
     )
 
+    ly_add_source_properties(
+        SOURCES Lib/Tests/Camera/test_EditorCamera.cpp
+        PROPERTY COMPILE_DEFINITIONS 
+        VALUES CAMERA_EDITOR_MODULE="$<TARGET_FILE_BASE_NAME:Camera.Editor>")
+
```

Comment:
cosmetic, put the `)` on the next line aligned with `ly_add_source_properties`

Reply:
Sure will do 👍

Reply:
`)` now on the next line